azure-quantum/tests/unit/test_qiskit.py

```diff
@@ -117,6 +117,16 @@ def test_plugins_estimate_cost_qiskit_ionq(self):
         cost = backend.estimate_cost(circuit, shots=100e3)
         assert np.round(cost.estimated_total) == 66.0
 
+        ## The following two tests are skipped until we can use a workspace
```

Comment:
optionally, we can add a new tag `ionq.aria` that is part of a custom capability so the tests are only run when this target is present in the workspace.

Reply:
Makes sense, but if we're planning to add Aria to the default test SKU, probably having a target specific tag will only have a short-lived effect, and that's not a common approach for others. If by next month Aria is not available, they I'll revisit this and add the tag, otherwise I'll just enable for `pytest.mark.ionq` in general.